addOns/reports/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed 
+- The traditional JSON Plus report now treats "otherinfo" as different per alert (Issue 7260).
```

Comment:
> - The Traditional JSON Plus report now treats "otherinfo" as different per alert (Issue 7260).

Reply:
The name of the report is `Traditional JSON Report with requests and responses` though.

Reply:
Yes, but that is only in the GUI, rest everywhere the name is `Traditional JSON Plus Report`. Should I change it ?

Reply:
Do you mean `traditional-json-plus`? That's the ID of the report not its name.

Reply:
Oh okay. Should I make any changes?

Reply:
It would be better to use the correct name.